__init__.py

```diff
@@ -1,7 +1,7 @@
 import json
 import logging
 
-import aiohttp
+import aiohttp, asyncio
```

Comment:
Please break the imports onto two lines. I appreciate this is a personal preference but it would bring it in line with the rest of the project.